app/src/main/java/com/asfoundation/wallet/topup/TopUpFragment.kt

```diff
@@ -301,16 +301,20 @@ class TopUpFragment : DaggerFragment(), TopUpFragmentView {
   override fun showLoading() {
     credit_card_info_container.visibility = View.GONE
     payment_methods.visibility = View.INVISIBLE
-    bonus_layout.visibility = View.GONE
-    bonus_msg.visibility = View.GONE
+    if (bonus_layout.visibility != View.GONE) {
```

Comment:
Same comment as above, regarding the possibility of having the INVISIBLE status already applied